forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -1128,29 +1489,38 @@ export const RecordsLog = ({
               </ConditionalComponent>
             </Grid> */}
             <Grid item xs={3}>
-              {isMinistryCoordinator ?
+              {isMinistryCoordinator ? (
                 <button
                   className={clsx("btn", "addAttachment", classes.createButton)}
                   variant="contained"
                   onClick={addAttachments}
                   color="primary"
-                  disabled = {conversionFormats?.length < 1}
+                  disabled={conversionFormats?.length < 1}
```

Comment:
didn't remove conversionFormats?.length < 1 because this is needed for the upload of documents.